src/DropDownMenu/DropDownMenu.js

```diff
@@ -220,7 +220,9 @@ class DropDownMenu extends Component {
   };
 
   handleItemTouchTap = (event, child, index) => {
-    this.props.onChange(event, index, child.props.value);
+    if (this.props.onChange !== undefined) {
+      this.props.onChange(event, index, child.props.value);
```

Comment:
I'm wondering if we shouldn't way the `setState` callback before calling this `onChange`.